packages/ra-ui-materialui/src/list/PaginationActions.js

```diff
@@ -163,16 +154,11 @@ export class PaginationActions extends Component {
 PaginationActions.propTypes = {
     backIconButtonProps: PropTypes.object,
     count: PropTypes.number.isRequired,
+    classes: PropTypes.object,
     nextIconButtonProps: PropTypes.object,
     onChangePage: PropTypes.func.isRequired,
     page: PropTypes.number.isRequired,
     rowsPerPage: PropTypes.number.isRequired,
 };
 
-const enhance = compose(
-    pure,
-    translate,
-    withStyles(styles)
-);
-
-export default enhance(PaginationActions);
+export default React.memo(PaginationActions, () => true);
```

Comment:
You're preventing this component to rerender when its props change. Please don't override the isEqual function (that's what `pure` does).